apps/full-site-editing/full-site-editing-plugin/full-site-editing/blocks/site-logo/index.php

```diff
@@ -0,0 +1,79 @@
+<?php
```

Comment:
I think this PHP file needs added here: https://github.com/Automattic/wp-calypso/blob/f3fd5a4f706bfa3498c9fa4a400a883d53750fc0/apps/full-site-editing/full-site-editing-plugin/full-site-editing-plugin.php#L46